src/renderer/src/helpers/DateHelper.ts

```diff
@@ -26,11 +35,15 @@ export class DateHelper {
     return `${year}${month}${day}`
   }
 
-  static format(date: DateArg<Date>, formatStr: string): string {
-    if (typeof date === 'number') {
-      date = date * 1000
+  static formatLocalized = (date: Date | string | number, options: TFormatLocalizedOptions): string => {
+    if (typeof date === 'string') {
+      date = new Date(date)
+    } else if (typeof date === 'number') {
+      date *= 1000
     }
 
-    return format(date, formatStr)
+    return dateFns.format(date, options.format, {
+      locale: DATE_FNS_LOCALE_BY_LANGUAGE_VALUE[options.language.value] || DATE_FNS_LOCALE_BY_LANGUAGE_VALUE['en'],
```

Comment:
I think `DATE_FNS_LOCALE_BY_LANGUAGE_VALUE['en']` isn't necessary. Language always exists.

What do you think?
```suggestion
      locale: DATE_FNS_LOCALE_BY_LANGUAGE_VALUE[options.language.value],
```